distdir_deps.bzl

```diff
@@ -351,45 +351,49 @@ DIST_DEPS = {
             "remotejdk11_linux_for_testing",
             "openjdk11_linux_archive",
         ],
-        "archive": "zulu11.50.19-ca-jdk11.0.12-linux_x64.tar.gz",
-        "sha256": "b8e8a63b79bc312aa90f3558edbea59e71495ef1a9c340e38900dd28a1c579f3",
-        "strip_prefix": "zulu11.50.19-ca-jdk11.0.12-linux_x64",
+        "archive": "zulu11.52.13-ca-jdk11.0.13-linux_x64.tar.gz",
+        "sha256": "77a126669b26b3a89e0117b0f28cddfcd24fcd7699b2c1d35f921487148b9a9f",
+        "strip_prefix": "zulu11.52.13-ca-jdk11.0.13-linux_x64",
         "urls": [
-            "https://mirror.bazel.build/openjdk/azul-zulu11.50.19-ca-jdk11.0.12/zulu11.50.19-ca-jdk11.0.12-linux_x64.tar.gz",
```

Comment:
The location in the mirror wasn't consistent. I updated it to use the relative path after `https://cdn.azul.com/` from the original URL, but this is easy to change if you have a preference